instrument/SXD_Definition_19_1.xml

```diff
@@ -0,0 +1,228 @@
+</component> 
+
+<component type="panel1346" idstart="12289" idfillbyfirst="y" idstepbyrow="64">
+<location x="-0.1369713215" y="0.000" z="0.1785045016" name="bank4" >
+  <rot val="-37.50000" axis-x="0" axis-y="1" axis-z="0">
+    <rot val="0.00000" axis-x="1" axis-y="0" axis-z="0" />
+  </rot>
+</location>
+<side-by-side-view-location x="0.85" y="0.7"/>
+</component> 
+
+<component type="panel" idstart="16385" idfillbyfirst="y" idstepbyrow="64">
+<location x="-0.225" y="0.000" z="0.0000" name="bank5" >
+  <rot val="90.00000" axis-x="0" axis-y="1" axis-z="0">
+    <rot val="0.00000" axis-x="1" axis-y="0" axis-z="0" />
+  </rot>
+</location>
+<side-by-side-view-location x="1.05" y="0.7"/>
+</component> 
+
+<component type="panel1346" idstart="20481" idfillbyfirst="y" idstepbyrow="64">
+<location x="-0.1369713215" y="0.000" z="-0.1785045016" name="bank6">
+  <rot val="-142.50000" axis-x="0" axis-y="1" axis-z="0">
+    <rot val="0.00000" axis-x="1" axis-y="0" axis-z="0" />
+  </rot>
+</location>
+<side-by-side-view-location x="1.25" y="0.7"/>
+</component> 
+
+<component type="panel" idstart="24577" idfillbyfirst="x" idstepbyrow="64">
+<location x="0.190919" y="-0.190919" z="0.000" name="bank7">
+  <rot val="90.00000" axis-x="0" axis-y="1" axis-z="0">
+    <rot val="45.00000" axis-x="1" axis-y="0" axis-z="0" />
+  </rot>
+</location>
+<side-by-side-view-location x="0.3" y="0.5"/>
+</component> 
+
+<component type="panel" idstart="28673" idfillbyfirst="x" idstepbyrow="64">
+<location x="0.000" y="-0.190919" z="0.190919" name="bank8">
+  <rot val="45.00000" axis-x="1" axis-y="0" axis-z="0" />
+</location>
+<side-by-side-view-location x="0.675" y="0.5"/>
+</component> 
+
+<component type="panel1346" idstart="32769" idfillbyfirst="x" idstepbyrow="64">
+<location x="-0.190919" y="-0.190919" z="0.000" name="bank9">
+  <rot val="90.00000" axis-x="0" axis-y="1" axis-z="0">
+    <rot val="-45.00000" axis-x="1" axis-y="0" axis-z="0" />
+ </rot>
+</location>
+<side-by-side-view-location x="1.05" y="0.5"/>
+</component> 
+
+<component type="panel1346" idstart="36865" idfillbyfirst="x" idstepbyrow="64">
+<location x="0.000" y="-0.190919" z="-0.190919" name="bank10">
+  <rot val="-45.00000" axis-x="1" axis-y="0" axis-z="0" />
+</location>
+<side-by-side-view-location x="1.425" y="0.5"/>
+</component> 
+
+<component type="panel11" idstart="40961" idfillbyfirst="x" idstepbyrow="64">
+<location x="0.000" y="-0.278" z="0.0000" name="bank11">
+  <rot val="90.00000" axis-x="1" axis-y="0" axis-z="0">
+    <rot val="0.00000" axis-x="1" axis-y="0" axis-z="0" />
+  </rot>
+</location>
+<side-by-side-view-location x="0.675" y="0.3"/>
+</component> 
+
+<!--
+<component type="panel" idstart="24601" idfillbyfirst="y" idstepbyrow="64">
+<location r="0.270" t="-90.00000" p="45.0000" name="bank7" rot="-90.0" axis-x="0" axis-y="1" axis-z="0">
+  <rot val="0.00000">
+    <rot val="45.00000" axis-x="1" axis-y="0" axis-z="0" />
+  </rot>
+</location>
+</component> 
```

Comment:
Not too familiar with instrument files, is it normal to leave lots of commented out stuff like this?

Reply:
Hmm I think it's not unusual but it doesn't mean it's good! This is just a copy from the original but as long as the commented out code is in the original I suppose there's no harm in deleting it from the `19_1` definition